app/src/main/proto/vss.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+option java_multiple_files = true;
+package org.vss;
+
+message GetObjectRequest {
+  /*
+    All APIs operate within a single storeId.
+    StoreId is keyspace identifier.
+    Ref: https://en.wikipedia.org/wiki/Keyspace_(distributed_data_store)
+    It is up to clients to use single or multiple store's for their usage.
+   */
+  string storeId = 1;
+  string key = 2;
+}
+message GetObjectResponse {
+  KeyValue value = 2;
+}
+
+message PutObjectRequest {
+  string storeId = 1;
+  /*
+    If present, write will only succeed if current DB globalVersion against a storeId is
+    same as in request.
+    Client is expected to global version against store. And PutRequest should contain
```

Comment:
Looks like there's a word missing in the sentence.

Reply:
Done, 👍 

Regarding above commend "The message docs can go into detail about the usage scenario. And fields can explain anything that should be considered when setting them."

I created an issue and we will add these details once we have implementation. 
https://github.com/lightningdevkit/vss-server/issues/3

Reply:
Could we do this now? Doing it in a follow-up just adds extra review burden.

Reply:
I have added lot more docs now for every field. But there are still many things which we could cover in here. 
I didn't want to stall this PR for docs changes and want to close on interface.